.github/workflows/galaxy-importer.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I suggest preparing the release in another PR where you also update the CHANGELOG.rst file. We use https://github.com/ansible-community/antsibull-changelog to generate the changelog. Since this is a first release, I suggest adding a changelogs/fragments/release_summary.yml with some information about the release and make sure that new modules are automatically added to the changelog. Also, if you are going with a 0.2.0 release, make sure that the version_added reflects within the modules.

Reply:
Ok, sure, that makes sense. Perhaps the title was misleading.